### DISCUSSION TITLE
Detect Mask Segmentation by prompt text generate new images.

### DISCUSSION THREAD

**Discussion by vlinh128** [lllyasviel/Fooocus#3345] [Q&A] [answered]
Version 2.5.0 includes a great mask segmentation feature, but currently, I need to go through two steps to generate new images for the "inpaint outpaint" model. The first step requires a "prompt_mask_text" to detect mask segmentations, and then the second step requires a "prompt_image_text" to generate the images. I wonder if it's possible to use just the "prompt_image_text" for both steps?

**Comment by mashb1t** [comment]
No, that doesn't make a lot of sense. One normally wants to replace something eith another thing, so if you only would provide the AI the target prompt it could not select what to detect properly.
What is your use ase for this and why do you think this would be beneficial to have?

**Reply by mashb1t** [reply, marked as answer]
=> https://github.com/lllyasviel/Fooocus/issues/3346#issuecomment-2244823874
